content/blog/managing-kubernetes-infrastructure-with-dotnet-and-pulumi/index.md

```diff
@@ -0,0 +1,251 @@
+---
+title: "Managing Kubernetes Infrastructure with .NET and Pulumi"
+authors: ["luke-hoban"]
+tags: ["dotnet","kubernetes"]
+date: "2019-12-04"
+meta_desc: "Manage Kubernetes with .NET using the Pulumi.Kubernetes resource provider for Pulumi."
+meta_image: "todo.png"
```

Comment:
Pending adding a proper meta image.

Reply:
made one - see if this works for you:
https://drive.google.com/file/d/1j-K1Xm7rwUjLK21RckDdzrSfykHfdmuF/view?usp=sharing